sound/soc/sdw_utils/soc_sdw_utils.c

```diff
@@ -1204,6 +1228,55 @@ int asoc_sdw_parse_sdw_endpoints(struct snd_soc_card *card,
 				    !(dai_info->quirk_exclude ^ !!(dai_info->quirk & ctx->mc_quirk)))
 					continue;
 
+				/* No need to check SDCA functions if there is only 1 endpoint present */
+				if (adr_dev->num_endpoints == 1)
```

Comment:
do we need a log entry here to state that only 1 endpoint is found and we assume it to be type X